pkg/client/options.go

```diff
@@ -116,6 +122,10 @@ func (dryRunAll) ApplyToPatch(opts *PatchOptions) {
 	opts.DryRun = []string{metav1.DryRunAll}
 }
 
+func (drun dryRunAll) ApplyToApply(opts *ApplyOptions) {
+	opts.DryRun = []string{metav1.DryRunAll}
+}
+
 // ApplyToPatch applies this configuration to the given delete options.
```

Comment:
```suggestion
// ApplyToDelete applies this configuration to the given delete options.
```
nit: or delete it (Intellij complained, sorry :))

Reply:
Done